Snowdog_Components/components/Organisms/footer/_footer.scss

```diff
@@ -164,49 +164,49 @@
     }
 
     &__social {
-        order: 1;
+        order: $footer__social-order;
 
         @include mq($screen-l) {
-            order: 0;
+            order: $footer__social-order\@large;
         }
     }
 
     &__social-handler {
-        display: flex;
-        flex-direction: column;
-        align-items: flex-start;
+        display: $footer__social-handler-display;
+        flex-direction: $footer__social-handler-flex-direction;
+        align-items: $footer__social-handler-align-items;
         margin: $footer__social-handler-margin;
 
         @include mq($screen-m) {
-            align-items: center;
-            flex-direction: row;
+            align-items: $footer__social-handler-align-items\@medium;
+            flex-direction: $footer__social-handler-flex-direction\@medium;
         }
 
         @include mq($screen-l) {
-            flex-direction: column;
-            flex: 1;
+            flex-direction: $footer__social-handler-flex-direction\@large;
+            flex: $footer__social-handler-flex\@large;
             margin: $footer__social-handler-margin\@large;
             padding: $footer__social-handler-padding\@large;
             border-bottom: $footer__newsletter-border;
-            position: relative;
+            position: $footer__social-handler-position\@large;
 
             &:before {
-                content: '';
-                position: absolute;
-                left: 0;
-                top: 0;
-                width: 1px;
-                height: 90%;
+                content: $footer__social-handler-before-content\@large;
```

Comment:
we can leave `content: ''` without a variable`. I don't this it will be ever changed